src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Security/TokenIntrospection.feature.disabled

```diff
@@ -0,0 +1,152 @@
+# This is a rough draft feature for future use. Will likely need modification
+# once we have determined more about how system interaction will work between
+# the DMS and the OAuth provider.
+
+Feature: JWT Token Introspection
+    Other than the Discovery endpoint, all HTTP requests against DMS endpoints
+    require a JSON Web Token (JWT) for authorizating the request. The DMS will
+    perform self-inspection to confirm the validity of the token, rather than
+    querying the OAuth provider.
+
+        Background:
+        # This might not be necessary - only keep it if other .feature files
+        # are somehow bypassing token authorization. # Might need to provide
+        # additional information here about # the allowed issuer and key
+        # information for validating the signature.
+            Given the Data Management Service must receive a token issued by "http://localhost"
+
+    Rule: Discovery API does not require an authorization token
```

Comment:
This is really nice, I think we can use it in the other feature files Esteban is working on